nf_core/launch.py

```diff
@@ -10,25 +11,32 @@
 import json
 import logging
 import os
-import PyInquirer
+import questionary
 import re
 import subprocess
 import textwrap
 import webbrowser
+from prompt_toolkit.styles import Style
```

Comment:
Even though this is a dependency of questionary, might still be good to add it to `setup.py` if we are importing it directly like this.

Reply:
It's directly imported by questionary anyway I think, as was an old version of it by PyInquirer.
But I guess it never hurts if it is directly in setup.py :)